plugin/forward/README.md

```diff
@@ -52,6 +52,7 @@ forward FROM TO... {
     tls_servername NAME
     policy random|round_robin|sequential
     health_check DURATION
+    health_check_non_recursive
```

Comment:
It's better/cleaner to make this an option property of `health_check`, probably something like
`healthcheck DURATION [no_rec]` or something

Reply:
Done. `health_check` now supports an optional `no_rec` argument.